docs/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 OutSystems
+Copyright (c) 2024 OutSystems
```

Comment:
We're releasing it under the Ionic naming so I believe this license it's not correct.
From my experience with creating the iOS repo, the Ionic License is inserted automatically when you create the repo and choose MIT.

Reply:
You're right. I updated the license to the Ionic one, can you check?